samples/matter/light_bulb/src/app_task.cpp

```diff
@@ -143,6 +160,9 @@ void AppTask::LightingActionEventHandler(const LightingEvent &event)
 	if (action == Nrf::PWMDevice::INVALID_ACTION || !Instance().mPWMDevice.InitiateAction(action, actor, NULL)) {
 		LOG_INF("An action could not be initiated.");
 	}
+#else
+	Nrf::GetBoard().GetLED(Nrf::DeviceLeds::LED2).Set(!Nrf::GetBoard().GetLED(Nrf::DeviceLeds::LED2).GetState());
```

Comment:
Are you sure that here should be a toggle? I guess it depends on `static_cast<Nrf::PWMDevice::Action_t>(event.Action)`, so depending on `event.Action` without casting to PWMDevice in this case.